helm_deploy/values-prod.yaml

```diff
@@ -13,6 +13,7 @@ generic-service:
     PRISON_API_ENDPOINT_URL: https://api.prison.service.justice.gov.uk
     FEATURE_ASYNC_ADJUDICATIONS: false
```

Comment:
@timoj58 when will you make these changes:

`
    FEATURE_ASYNC_ADJUDICATIONS: true
    FEATURE_CHARGE_NUMBER: true
    FEATURE_NOMIS_SOURCE_OF_TRUTH_SUMMARY: true
    FEATURE_SKIP_EXISTING: false
    FEATURE_ASYNC_HEARINGS: true
    FEATURE_ASYNC_PUNISHMENTS: true
    FEATURE_ASYNC_OUTCOMES: true
`